Misc/NEWS.d/next/Security/2020-06-21-19-01-01.bpo-41004.P6i7Nj.rst

```diff
@@ -0,0 +1,2 @@
+The hash() methods of classes IPv4Interface and IPv6Interface (of the ipaddress library) had issue of generating constant hash values of 32 and 128 respectively causing hash collisions.
+The fix uses the hash() function to generate hash values for the objects instead of XOR operation
```

Comment:
Please put these two sentences on the same line, without a line break between them. Also, you should mention that it's using `hash()` to compute a hash of a tuple of the address, prefix length, and the network address. You don't need to mention the old XOR behavior.